pkgs/development/python-modules/torch/default.nix

```diff
@@ -184,6 +173,13 @@ in buildPythonPackage rec {
       --replace "set(ROCM_PATH \$ENV{ROCM_PATH})" \
         "set(ROCM_PATH \$ENV{ROCM_PATH})''\nset(ROCM_VERSION ${lib.concatStrings (lib.intersperse "0" (lib.splitString "." hip.version))})"
   ''
+  # Detection of NCCL version doesn't work particularly well when using the static binary.
```

Comment:
not necessarily a blocker but should we report this upstream

Reply:
We should, but I haven't had the time to package it up properly into an issue :(

Reply:
Tracked in https://github.com/NixOS/nixpkgs/issues/255876 (need to verify that this isn't caused by Nixpkgs before reporting upstream).